src/commands/community/prune/inactive.ts

```diff
@@ -33,21 +34,29 @@ export async function pruneInactiveExecute(i: ButtonInteraction) {
 const command: Command = {
   id: "prune_inactive",
   command: "inactive",
-  brief: "Remove 30-day inactive users",
+  brief: "Remove users with specific inactive days",
   category: "Community",
   run: async (msg) => {
     if (!msg.guild) {
       throw new GuildIdNotFoundError({ message: msg })
     }
 
-    const pruned = await msg.guild?.members.prune({ dry: true, days: 30 })
+    const args = getCommandArguments(msg)
+    if (args.length < 3 || parseInt(args[2]) < 5) {
+      throw new CommandError({
+        message: msg,
+        description: "Inactive days should be a number from 5 and higher",
+      })
+    }
+    const days = parseInt(args[2])
```

Comment:
```suggestion
    const days = parseInt(args[2], 10)
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

<img width="786" alt="CleanShot 2022-10-21 at 10 36 06@2x" src="https://user-images.githubusercontent.com/25856620/197105852-6d1882c3-0596-46e8-af1b-1d2e068adb01.png">